ui/src/app/views/bug-report/component.tsx

```diff
@@ -0,0 +1,139 @@
+import {Component, Input} from '@angular/core';
+
+import {bugReportApi} from 'app/services/swagger-fetch-clients';
+import {reactStyles, ReactWrapperBase, withUserProfile} from 'app/utils/index';
+
+import {AlertDanger} from 'app/components/alert';
+import {Button} from 'app/components/buttons';
+import {ClrIcon} from 'app/components/icons';
+import {Modal, ModalBody, ModalFooter, ModalInput, ModalTitle} from 'app/components/modals';
+import {
+  BugReport,
+  BugReportType,
+  Profile
+} from 'generated/fetch';
+import * as React from 'react';
```

Comment:
Nit: I'd bump library imports to the top, and separate out the generated into its own import block.